docs/conf/portland/2021/news/announcing-speakers.rst

```diff
@@ -0,0 +1,55 @@
+:template: {{year}}/generic.html
+
+.. post:: February 14, 2021
+   :tags: portland-2021, speakers, tickets, visiting, sponsors
+
+Announcing speakers and sessions
+================================
+
+We're writing today with an update on the fast-approaching Portland conference.
+
+First off, we're delighted to announce that we've finalized our speakers for the event.
```

Comment:
We aren't doing anything other than this first, so feels like a weird transition. Also probably good to mention the other new items in the head. 

```suggestion
We're delighted to announce that we've finalized our speakers for the event and have t-shirts on sale.
```